openfisca_survey_manager/variables.py

```diff
@@ -2,12 +2,35 @@
 from numpy import arange
 
 from openfisca_core.model_api import where
+from openfisca_france.model.base import *
 from openfisca_survey_manager.statshelpers import (
     mark_weighted_percentiles,
     weightedcalcs_quantiles
     )
 
 
+def create_quantile(x, nquantiles, weight_variable, entity_name):
+
+    class quantile(Variable):
+        value_type = int
+        entity = entity_name
+        label = u"Quantile"
```

Comment:
```suggestion
        label = "Quantile"
```

Unicode mark is nor more needed